packages/cardano-services/src/ChainHistory/openApi.json

```diff
@@ -361,7 +361,8 @@
               "spend",
               "mint",
               "certificate",
-              "withdrawal"
+              "withdrawal",
+              "representative"
```

Comment:
I would expect `representative` to be in the switch statement above. Maybe this is something else

Reply:
Nice catch, 'representative' is actually gone, but it was missing other conway redeemer properties https://github.com/input-output-hk/cardano-js-sdk/pull/1176/commits/3ca4d2724dd90ef663afa637d188d9410acb69dc

This is probably an error when rebasing after merge from conway-era :sweat_smile: